sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/Generated/Models/Type.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.CosmosDB.Models
+{
+    /// <summary> Type of the connection string. </summary>
+    public readonly partial struct Type : IEquatable<Type>
```

Comment:
`Type` is too generic, consider renaming to `CosmosDBConnectionStringType`.

Reply:
Renamed